hipchat.go

```diff
@@ -86,15 +86,22 @@ type ErrorResponse struct {
 type Client struct {
 	AuthToken string
 	BaseURL   string
+	ProxyURL  string
```

Comment:
can you be more specific about your comment ? do you want to remove duplicate code  and implement single client for all methods (PostMessage, ListRoom) ? or do you wann specify proxy timeout and round-tripper settings ?